src/Controller/ListController.php

```diff
@@ -96,4 +97,19 @@ public function getMembersAction(Request $request, SubscriberList $list): View
 
         return View::create()->setData($list->getSubscribers());
     }
+
+    /**
+     * Gets the total number of subscribers of a list.
+     *
+     * @param Request $request
+     * @param SubscriberList $list
+     *
+     * @return View
+     */
+    public function getCountAction(Request $request, SubscriberList $list): View
```

Comment:
I'd prefer a different, more specific name. Currently, it "for this list, count something", and that "something" is not specified. I propose `getCountSubscribersAction`. What do you think?

Reply:
@oliverklee I agree.